executor/distsql.go

```diff
@@ -484,9 +487,9 @@ func (e *IndexLookUpExecutor) startIndexWorker(ctx context.Context, kvRanges []k
 		}
 		if e.runtimeStats != nil {
 			copStats := e.ctx.GetSessionVars().StmtCtx.RuntimeStatsColl.GetRootStats(e.idxPlans[len(e.idxPlans)-1].ExplainID().String())
-			copStats.SetRowNum(count)
+			copStats.SetRowNum(int64(count))
```

Comment:
Not related to this PR, but it seems really strange to use `int64` instead of `uint64` for a `count`.

Reply:
@qw4990 Is there any specific reason why we are using `int`/`int64` for `SetRequiredRows`/`SetRowNum` respectively?

Reply:
For `RowNum`, we use methods like `atomic.XXXInt64` to maintain it so its type is `int64`.
And for `RequiredRows` we maintain it using Golang statement and for convenience, we set its type to `int`.